src/AccordionItem/accordion-item.js

```diff
@@ -30,23 +30,26 @@ class AccordionItem extends Component {
         const { itemUuid } = this.state;
 
         return React.Children.map(children, (item) => {
-            const itemProps = {
-                expanded: expanded,
-            };
+            const itemProps = {};
 
-            if (item.type.accordionElementName === 'AccordionItemTitle') {
-                itemProps.key = 'title';
-                itemProps.id = `accordion__title-${itemUuid}`;
-                itemProps.ariaControls = `accordion__body-${itemUuid}`;
-                itemProps.onClick = onClick;
-                itemProps.role = accordion ? 'tab' : 'button';
-            } else if (item.type.accordionElementName === 'AccordionItemBody') {
-                itemProps.key = 'body';
-                itemProps.id = `accordion__body-${itemUuid}`;
-                itemProps.role = accordion ? 'tabpanel' : '';
+            if (item.type.accordionElementName === 'AccordionItemTitle' ||
+                item.type.accordionElementName === 'AccordionItemBody') {
+                itemProps.expanded = expanded;
+                if (item.type.accordionElementName === 'AccordionItemTitle') {    
+                    itemProps.key = 'title';
+                    itemProps.id = `accordion__title-${itemUuid}`;
+                    itemProps.ariaControls = `accordion__body-${itemUuid}`;
+                    itemProps.onClick = onClick;
+                    itemProps.role = accordion ? 'tab' : 'button';
+                } else { // AccordionItemBody
+                    itemProps.key = 'body';
+                    itemProps.id = `accordion__body-${itemUuid}`;
+                    itemProps.role = accordion ? 'tabpanel' : '';
+                }
```

Comment:
The nested `if/else` is a bit confusing, I think it would be clearer to have it separated like before even if there is some redundancy.